library/firewall_lib.py

```diff
@@ -39,6 +39,12 @@
   Manage firewall with firewalld on Fedora and RHEL-7+.
 author: "Thomas Woerner (@t-woerner)"
 options:
+  restore_default_settings:
```

Comment:
I think this should be `previous: replaced` based on https://linux-system-roles.github.io/documentation/incremental_settings.html "Replacing an entire list of items with the given values with previous: replaced"